### PR TITLE
Candidate election expense importer + Django backend

### PR DESCRIPTION
**Pull request by jlehtoma**

### CODE REVIEW

File: importers/backends/django.py [hunk 1]
```diff
@@ -314,3 +314,139 @@ def submit_candidates(self, election, candidates):
             count += 1
 
         self.logger.info("%d candidates updated" % count)
+        
+    def submit_prebudgets(self, expenses, expense_types):
```
Comment (juyrjola) [inline importers/backends/django.py:318]: Include information about election here

File: importers/backends/django.py [hunk 2]
```diff
@@ -314,3 +314,139 @@ def submit_candidates(self, election, candidates):
...
+            
+            try:
+                # Construct a String to describe person at hand (candidate)
+                person_str = "%s %s (%s)" % (candidate_expenses['first_names'].encode('utf8') ,
+                                             candidate_expenses['last_name'].encode('utf8') ,
+                                             candidate_expenses['municipality'] )
+                
+                # Municipality given as String, do a lookup for an integer id.
+                # Needed to identify the candidate
+                municipality_id = Municipality.objects.get(name=candidate_expenses['municipality']).id
```
Comment (juyrjola) [inline importers/backends/django.py]: Possibly use the object directly

File: importers/backends/django.py [hunk 3]
```diff
@@ -314,3 +314,139 @@ def submit_candidates(self, election, candidates):
...
+        
+        for candidate_expenses in expenses:
+            
+            # Keep track if the candidate expenses were updated
+            updated = False
+            
+            db.reset_queries()
+            
+            # Doest the candidate have any actual expenses (with numbers)
+            types = [etype['type'] for etype in expense_types]
```
Comment (juyrjola) [inline importers/backends/django.py]: move this outside the candidate loop

File: importers/backends/django.py (hunk 1 above)
Comment (jlehtoma) [inline importers/backends/django.py:318]: The election information would also need to be accounted in the Expense table, right?

File: importers/backends/django.py (hunk 2 above)
Comment (jlehtoma) [inline importers/backends/django.py]: Done

File: importers/backends/django.py (hunk 3 above)
Comment (jlehtoma) [inline importers/backends/django.py]: Done

File: importers/backends/django.py (hunk 1 above)
Comment (jlehtoma) [inline importers/backends/django.py:318]: Sorry, my bad. Dicts in expenses list only need to include {'type': 'muni', 'year':2012}   in order to identify candidate.

**Pull request closed without merging**